beebop/app.py

```diff
@@ -56,6 +56,16 @@ def check_connection(redis):
 
 
 def generate_zip(path_folder, type, cluster):
+    """
+    This generates a .zip folder with results data.
+
+    Arguments:
+    path_folder - folder to be zipped
+    type - can be either 'microreact' or 'network'
+    cluster - only relevant for 'network', since there are multiple
```

Comment:
What sort of thing is `cluster`? Is it the name of a cluster? If not relevant should it be None?